lambda-buffers-proto/compiler.proto

```diff
@@ -636,45 +636,56 @@ message ProtoParseError {
 // Kind checking errors.
 message KindCheckError {
 
-  // Error referring to an unbound term. This usually means that the term was
-  // not defined.
-  message UnboundTermError {
-    TyName ty_name = 1;
-    VarName var_name = 2;
+  // Unbound variable ty_var detected in term ty_def.
+  message UnboundTyVarError {
+    TyDef ty_def = 1;
+    TyVar ty_var = 2;
+    ModuleName module_name = 3;
```

Comment:
Sorry mate for such a nit, but the module_name goes first as that's what we've being doing before. Would you mind putting it above?